PowerUp/app/src/main/java/powerup/systers/com/data/entities/Answer.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * @desc this class holds getter and setter methods for each question/answer pair
+ * each answer has a specific point value
+ * examples include setAnswerID() and getPoints()
+ */
+
+package powerup.systers.com.data.entities;
+
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+
+@Entity(tableName = "Answer")
+public class Answer {
+
+    @PrimaryKey
+    @ColumnInfo(name = "AID")
+    private Integer answerID;
+
+    @ColumnInfo(name = "ADes")
+    private String answerDescription;
+
+    @ColumnInfo(name = "QID")
+    private Integer questionID;
+
+    @ColumnInfo(name = "NextID")
+    private Integer nextQuestionID;
+
+    @ColumnInfo(name = "Points")
```

Comment:
Why are you using the `ColumnInfo` annotation when the variable names are same? Also why change the column names at all?

Reply:
I chose the variables names according to java guidelines randomly. Though, Column names are  taken from PowerupContract File. Shall I use variables names as column names?

Reply:
By default, the column name is the variable name. If you want the column name to be different then use the `@ColumnInfo` annotation.

I don't see why we need to use the `@ColumnInfo` annotation.

Reply:
I'll update the PR: remove columninfo annotation & let it use by default variable name as the column name.